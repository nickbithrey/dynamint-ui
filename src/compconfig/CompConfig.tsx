import * as React from 'react';
import { Route, Switch } from 'react-router-dom';
import Selection from './selection';
import Details from './details';

const CompConfig = () => (
    <Switch>
        <Route exact path={'/compconfig'} component={Selection} />
        <Route exact path={'/compconfig/details'} component={Details} />
    </Switch>
);

export default CompConfig;