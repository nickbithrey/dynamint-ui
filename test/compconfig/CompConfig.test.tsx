import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import CompConfig from '~/compconfig/CompConfig';

describe('<CompConfig />', () => {
    
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<CompConfig />);
    });
    
    it('renders a valid tag and matches snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
    
});