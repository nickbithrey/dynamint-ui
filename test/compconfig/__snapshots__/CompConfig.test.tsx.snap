// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CompConfig /> renders a valid tag and matches snapshot 1`] = `
<Switch>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Loading)",
        "type": [Function],
      }
    }
    exact={true}
    path="/compconfig"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Loading)",
        "type": [Function],
      }
    }
    exact={true}
    path="/compconfig/details"
  />
</Switch>
`;
